Description: use system icons
 Since v4.1, upstream ships icons having the Reshot license which is not DFSG
 compliant.
 .
 This patch uses the system icons instead, as czkawka did in v4.0
Origin: https://github.com/qarmin/czkawka/pull/659
Bug: https://github.com/qarmin/czkawka/issues/1029

--- a/src/gui_structs/gui_data.rs
+++ b/src/gui_structs/gui_data.rs
@@ -35,7 +35,7 @@
 use crate::taskbar_progress::TaskbarProgress;
 
 pub const ICON_ABOUT: &[u8] = include_bytes!("../../icons/icon_about.png");
-pub const CZK_ICON_ADD: &[u8] = include_bytes!("../../icons/czk_add.svg");
+/*pub const CZK_ICON_ADD: &[u8] = include_bytes!("../../icons/czk_add.svg");
 pub const CZK_ICON_COMPARE: &[u8] = include_bytes!("../../icons/czk_compare.svg");
 pub const CZK_ICON_DELETE: &[u8] = include_bytes!("../../icons/czk_delete.svg");
 pub const CZK_ICON_HARDLINK: &[u8] = include_bytes!("../../icons/czk_hardlink.svg");
@@ -54,7 +54,7 @@
 pub const CZK_ICON_STOP: &[u8] = include_bytes!("../../icons/czk_stop.svg");
 pub const CZK_ICON_SYMLINK: &[u8] = include_bytes!("../../icons/czk_symlink.svg");
 pub const CZK_ICON_TRASH: &[u8] = include_bytes!("../../icons/czk_trash.svg");
-
+*/
 #[derive(Clone)]
 pub struct GuiData {
     // Windows
--- a/src/gui_structs/gui_bottom_buttons.rs
+++ b/src/gui_structs/gui_bottom_buttons.rs
@@ -1,11 +1,11 @@
 use gtk4::prelude::*;
 use gtk4::{GestureClick, Widget};
 
-use crate::gui_structs::gui_data::CZK_ICON_SORT;
+//use crate::gui_structs::gui_data::CZK_ICON_SORT;
 use crate::help_functions::{get_custom_label_from_widget, set_icon_of_button, BottomButtonsEnum};
 use crate::{
-    flg, CZK_ICON_COMPARE, CZK_ICON_HARDLINK, CZK_ICON_HIDE_DOWN, CZK_ICON_HIDE_UP, CZK_ICON_MOVE, CZK_ICON_SAVE, CZK_ICON_SEARCH, CZK_ICON_SELECT, CZK_ICON_SYMLINK,
-    CZK_ICON_TRASH,
+    flg,// CZK_ICON_COMPARE, CZK_ICON_HARDLINK, CZK_ICON_HIDE_DOWN, CZK_ICON_HIDE_UP, CZK_ICON_MOVE, CZK_ICON_SAVE, CZK_ICON_SEARCH, CZK_ICON_SELECT, CZK_ICON_SYMLINK,
+    //CZK_ICON_TRASH,
 };
 
 #[derive(Clone)]
@@ -56,7 +56,7 @@
         buttons_select.add_controller(gc_buttons_select.clone());
         buttons_sort.add_controller(gc_buttons_sort.clone());
 
-        set_icon_of_button(&buttons_search, CZK_ICON_SEARCH);
+        /*set_icon_of_button(&buttons_search, CZK_ICON_SEARCH);
         set_icon_of_button(&buttons_select, CZK_ICON_SELECT);
         set_icon_of_button(&buttons_delete, CZK_ICON_TRASH);
         set_icon_of_button(&buttons_save, CZK_ICON_SAVE);
@@ -66,7 +66,7 @@
         set_icon_of_button(&buttons_compare, CZK_ICON_COMPARE);
         set_icon_of_button(&buttons_sort, CZK_ICON_SORT);
         set_icon_of_button(&buttons_show_errors, CZK_ICON_HIDE_DOWN);
-        set_icon_of_button(&buttons_show_upper_notebook, CZK_ICON_HIDE_UP);
+        set_icon_of_button(&buttons_show_upper_notebook, CZK_ICON_HIDE_UP);*/
 
         let buttons_names = [
             BottomButtonsEnum::Search,
--- a/src/gui_structs/gui_compare_images.rs
+++ b/src/gui_structs/gui_compare_images.rs
@@ -5,7 +5,7 @@
 use gtk4::{Builder, TreePath};
 
 use crate::help_functions::set_icon_of_button;
-use crate::{flg, CZK_ICON_LEFT, CZK_ICON_RIGHT};
+use crate::{flg};
 
 #[derive(Clone)]
 pub struct GuiCompareImages {
@@ -60,8 +60,8 @@
         let shared_image_cache = Rc::new(RefCell::new(Vec::new()));
         let shared_using_for_preview = Rc::new(RefCell::new((None, None)));
 
-        set_icon_of_button(&button_go_previous_compare_group, CZK_ICON_LEFT);
-        set_icon_of_button(&button_go_next_compare_group, CZK_ICON_RIGHT);
+        //set_icon_of_button(&button_go_previous_compare_group, CZK_ICON_LEFT);
+        //set_icon_of_button(&button_go_next_compare_group, CZK_ICON_RIGHT);
 
         Self {
             window_compare,
--- a/src/gui_structs/gui_header.rs
+++ b/src/gui_structs/gui_header.rs
@@ -1,7 +1,7 @@
 use gtk4::prelude::*;
 
 use crate::help_functions::set_icon_of_button;
-use crate::{flg, CZK_ICON_INFO, CZK_ICON_SETTINGS};
+use crate::{flg};
 
 #[derive(Clone)]
 pub struct GuiHeader {
@@ -14,8 +14,8 @@
         let button_settings: gtk4::Button = builder.object("button_settings").expect("Cambalache");
         let button_app_info: gtk4::Button = builder.object("button_app_info").expect("Cambalache");
 
-        set_icon_of_button(&button_settings, CZK_ICON_SETTINGS);
-        set_icon_of_button(&button_app_info, CZK_ICON_INFO);
+        //set_icon_of_button(&button_settings, CZK_ICON_SETTINGS);
+        //set_icon_of_button(&button_app_info, CZK_ICON_INFO);
 
         Self { button_settings, button_app_info }
     }
--- a/src/gui_structs/gui_progress_dialog.rs
+++ b/src/gui_structs/gui_progress_dialog.rs
@@ -2,7 +2,7 @@
 use gtk4::{Builder, EventControllerKey, Window};
 
 use crate::help_functions::{get_custom_label_from_widget, set_icon_of_button};
-use crate::{flg, CZK_ICON_STOP};
+use crate::{flg};
 
 #[derive(Clone)]
 pub struct GuiProgressDialog {
@@ -44,7 +44,7 @@
         let evk_button_stop_in_dialog = EventControllerKey::new();
         button_stop_in_dialog.add_controller(evk_button_stop_in_dialog.clone());
 
-        set_icon_of_button(&button_stop_in_dialog, CZK_ICON_STOP);
+        //set_icon_of_button(&button_stop_in_dialog, CZK_ICON_STOP);
 
         Self {
             window_progress,
--- a/src/gui_structs/gui_upper_notebook.rs
+++ b/src/gui_structs/gui_upper_notebook.rs
@@ -3,7 +3,7 @@
 
 use crate::help_functions::{get_all_direct_children, get_custom_label_from_widget, set_icon_of_button};
 use crate::notebook_enums::NotebookUpperEnum;
-use crate::{flg, CZK_ICON_ADD, CZK_ICON_DELETE, CZK_ICON_MANUAL_ADD};
+use crate::{flg};
 
 #[derive(Clone)]
 pub struct GuiUpperNotebook {
@@ -88,12 +88,12 @@
         let label_general_min_size: gtk4::Label = builder.object("label_general_min_size").expect("Cambalache");
         let label_general_max_size: gtk4::Label = builder.object("label_general_max_size").expect("Cambalache");
 
-        set_icon_of_button(&buttons_add_included_directory, CZK_ICON_ADD);
+        /*set_icon_of_button(&buttons_add_included_directory, CZK_ICON_ADD);
         set_icon_of_button(&buttons_manual_add_included_directory, CZK_ICON_MANUAL_ADD);
         set_icon_of_button(&buttons_remove_included_directory, CZK_ICON_DELETE);
         set_icon_of_button(&buttons_add_excluded_directory, CZK_ICON_ADD);
         set_icon_of_button(&buttons_manual_add_excluded_directory, CZK_ICON_MANUAL_ADD);
-        set_icon_of_button(&buttons_remove_excluded_directory, CZK_ICON_DELETE);
+        set_icon_of_button(&buttons_remove_excluded_directory, CZK_ICON_DELETE);*/
 
         Self {
             notebook_upper,
--- a/ui/compare_images.ui
+++ b/ui/compare_images.ui
@@ -16,7 +16,7 @@
                 <property name="receives-default">1</property>
                 <child>
                   <object class="GtkImage">
-                    <property name="icon-name">image-missing</property>
+                    <property name="icon-name">go-previous</property>
                   </object>
                 </child>
               </object>
@@ -34,7 +34,7 @@
                 <property name="receives-default">1</property>
                 <child>
                   <object class="GtkImage">
-                    <property name="icon-name">image-missing</property>
+                    <property name="icon-name">go-next</property>
                   </object>
                 </child>
               </object>
--- a/ui/main_window.ui
+++ b/ui/main_window.ui
@@ -47,7 +47,7 @@
                                     <property name="spacing">4</property>
                                     <child>
                                       <object class="GtkImage">
-                                        <property name="icon-name">image-missing</property>
+                                        <property name="icon-name">list-add</property>
                                       </object>
                                     </child>
                                     <child>
@@ -69,7 +69,7 @@
                                     <property name="spacing">4</property>
                                     <child>
                                       <object class="GtkImage">
-                                        <property name="icon-name">image-missing</property>
+                                        <property name="icon-name">list-remove</property>
                                       </object>
                                     </child>
                                     <child>
@@ -92,7 +92,7 @@
                                     <property name="spacing">4</property>
                                     <child>
                                       <object class="GtkImage">
-                                        <property name="icon-name">image-missing</property>
+                                        <property name="icon-name">insert-link</property>
                                       </object>
                                     </child>
                                     <child>
@@ -153,7 +153,7 @@
                                     <property name="spacing">4</property>
                                     <child>
                                       <object class="GtkImage">
-                                        <property name="icon-name">image-missing</property>
+                                        <property name="icon-name">list-add</property>
                                       </object>
                                     </child>
                                     <child>
@@ -175,7 +175,7 @@
                                     <property name="spacing">4</property>
                                     <child>
                                       <object class="GtkImage">
-                                        <property name="icon-name">image-missing</property>
+                                        <property name="icon-name">list-remove</property>
                                       </object>
                                     </child>
                                     <child>
@@ -198,7 +198,7 @@
                                     <property name="spacing">4</property>
                                     <child>
                                       <object class="GtkImage">
-                                        <property name="icon-name">image-missing</property>
+                                        <property name="icon-name">insert-link</property>
                                       </object>
                                     </child>
                                     <child>
@@ -917,7 +917,7 @@
                         <property name="spacing">2</property>
                         <child>
                           <object class="GtkImage">
-                            <property name="icon-name">image-missing</property>
+                            <property name="icon-name">edit-find</property>
                           </object>
                         </child>
                         <child>
@@ -950,7 +950,7 @@
                             <property name="spacing">2</property>
                             <child>
                               <object class="GtkImage">
-                                <property name="icon-name">image-missing</property>
+                                <property name="icon-name">edit-select-all</property>
                               </object>
                             </child>
                             <child>
@@ -973,7 +973,7 @@
                             <property name="spacing">2</property>
                             <child>
                               <object class="GtkImage">
-                                <property name="icon-name">image-missing</property>
+                                <property name="icon-name">mail-send-receive</property>
                               </object>
                             </child>
                             <child>
@@ -995,7 +995,7 @@
                             <property name="spacing">2</property>
                             <child>
                               <object class="GtkImage">
-                                <property name="icon-name">image-missing</property>
+                                <property name="icon-name">video-display</property>
                               </object>
                             </child>
                             <child>
@@ -1017,7 +1017,7 @@
                             <property name="spacing">2</property>
                             <child>
                               <object class="GtkImage">
-                                <property name="icon-name">image-missing</property>
+                                <property name="icon-name">list-remove</property>
                               </object>
                             </child>
                             <child>
@@ -1039,7 +1039,7 @@
                             <property name="spacing">2</property>
                             <child>
                               <object class="GtkImage">
-                                <property name="icon-name">image-missing</property>
+                                <property name="icon-name">folder</property>
                               </object>
                             </child>
                             <child>
@@ -1061,7 +1061,7 @@
                             <property name="spacing">2</property>
                             <child>
                               <object class="GtkImage">
-                                <property name="icon-name">image-missing</property>
+                                <property name="icon-name">document-save</property>
                               </object>
                             </child>
                             <child>
@@ -1083,7 +1083,7 @@
                             <property name="spacing">2</property>
                             <child>
                               <object class="GtkImage">
-                                <property name="icon-name">image-missing</property>
+                                <property name="icon-name">network-transmit-receive</property>
                               </object>
                             </child>
                             <child>
@@ -1105,7 +1105,7 @@
                             <property name="spacing">2</property>
                             <child>
                               <object class="GtkImage">
-                                <property name="icon-name">image-missing</property>
+                                <property name="icon-name">text-x-generic-template</property>
                               </object>
                             </child>
                             <child>
@@ -1126,7 +1126,7 @@
                     <child>
                       <object class="GtkImage">
                         <property name="halign">center</property>
-                        <property name="icon-name">image-missing</property>
+                        <property name="icon-name">format-justify-fill</property>
                       </object>
                     </child>
                   </object>
@@ -1138,7 +1138,7 @@
                     <child>
                       <object class="GtkImage">
                         <property name="halign">center</property>
-                        <property name="icon-name">image-missing</property>
+                        <property name="icon-name">utilities-system-monitor</property>
                       </object>
                     </child>
                   </object>
@@ -1200,7 +1200,7 @@
                 <property name="receives-default">1</property>
                 <child>
                   <object class="GtkImage">
-                    <property name="icon-name">image-missing</property>
+                    <property name="icon-name">applications-engineering</property>
                   </object>
                 </child>
               </object>
@@ -1211,7 +1211,7 @@
                 <property name="receives-default">1</property>
                 <child>
                   <object class="GtkImage">
-                    <property name="icon-name">image-missing</property>
+                    <property name="icon-name">dialog-information</property>
                   </object>
                 </child>
               </object>
--- a/ui/progress.ui
+++ b/ui/progress.ui
@@ -77,7 +77,7 @@
               <object class="GtkBox">
                 <child>
                   <object class="GtkImage">
-                    <property name="icon-name">image-missing</property>
+                    <property name="icon-name">dialog-cancel</property>
                   </object>
                 </child>
                 <child>
